kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -82,6 +82,15 @@
                   :userType="getUserKind"
                 />
               </div>
+              <div v-if="getUserKind === 'learner'">
```

Comment:
At some point (but not necessarily now) we should refactor the name of this getter to `userKind` - calling it `getUserKind` is more in line with our practice of getters that are actually functions (I had to double check that this was not expecting the id of a user as an argument).